web/index.html

```diff
@@ -31,7 +31,10 @@
         #root > div > div {
             height: 100% !important;
         }
-        :root {
+        .scrollbar-light {
+            color-scheme: light !important;
+        }
+        .scrollbar-light {
```

Comment:
I think one of these should be scrollbar-dark no?

Reply:
actually, forgot to remove this. This is obsolete